supervisor/supervisorctl.py

```diff
@@ -983,6 +983,11 @@ def help_shutdown(self):
         self.ctl.output("shutdown \tShut the remote supervisord down.")
 
     def do_reload(self, arg):
+        if arg:
+            self.ctl.output('Error: reload given with a process name. Maybe you meant restart?')
```

Comment:
This should use `handle_error` [like the other](https://github.com/Supervisor/supervisor/blob/f4c55bacd502d46652444cc20405121a75d0762b/supervisor/supervisorctl.py#L831-L832) argument checks in this file.